packages/mui-base/src/TablePaginationUnstyled/TablePaginationActionsUnstyled.tsx

```diff
@@ -0,0 +1,128 @@
+import * as React from 'react';
+import { OverridableComponent } from '@mui/types';
+import { appendOwnerState } from '../utils';
+import { ItemAriaLabelType } from './TablePaginationUnstyledProps';
+import TablePaginationActionsUnstyledProps, {
+  TablePaginationActionsUnstyledTypeMap,
+} from './TablePaginationActionsUnstyledProps';
+
+const LastPageIconDefault = () => <span>{'>|'}</span>;
+const FirstPageIconDefault = () => <span>{'|<'}</span>;
+const NextPageIconDefault = () => <span>{'>'}</span>;
+const BackPageIconDefault = () => <span>{'<'}</span>;
```

Comment:
Could we use some better-looking Unicode arrows?
⇤ ← → ⇥

Reply:
Sure 👍

Reply:
What do you think of these:
![image](https://user-images.githubusercontent.com/4512430/144063754-cc58022b-8fa5-4ae2-ae02-8209778d5046.png)

The ones you propose have different height, and the buttons look broken:
![image](https://user-images.githubusercontent.com/4512430/144063886-5729fbc8-6daa-4bb3-aac3-a9ff4fa83c6f.png)

Reply:
> What do you think of these

Yep, they look good!